dataplane/redbeams/redbeams.go

```diff
@@ -53,14 +52,120 @@ func ListRBDMSInstances(c *cli.Context) {
 	output.WriteList(listHeader, tableRows)
 }
 
-func DeleteRBDMSInstance(c *cli.Context) {
-	defer commonutils.TimeTrack(time.Now(), "delete RBDMS instance")
+func GetDatabaseServer(c *cli.Context) {
+	defer commonutils.TimeTrack(time.Now(), "Get a database server by CRN")
 	crn := c.String(fl.FlCrn.Name)
+	redbeamsDbServerClient := ClientRedbeams(*oauth.NewRedbeamsClientFromContext(c)).Redbeams.V4databaseservers
+
+	log.Infof("[GetDBServer] Getting database server with CRN: %s", crn)
+	resp, err := redbeamsDbServerClient.GetDatabaseServerByCrn(v4databaseservers.NewGetDatabaseServerByCrnParams().WithCrn(crn))
+	if err != nil {
+		commonutils.LogErrorAndExit(err)
+	}
+
+	server := resp.Payload
+	output := commonutils.Output{Format: c.String(fl.FlOutputOptional.Name)}
+	output.Write(listHeader, &redbeamsDetails{
+		Name:           *server.Name,
+		CRN:            server.Crn,
+		EnvironmentCrn: *server.EnvironmentCrn,
+	})
```

Comment:
I would add status here, as well. 
Didn't you want to print all the other specs for database server, like host, port etc?

Reply:
Yep, I'm adding it. Turns out that I needed to regenerate the client code; the response object was missing the status, because it was generated so long ago.